### ISSUE TITLE
bin/hsqldb listens on all interfaces by default

### ISSUE THREAD

**Issue by linnea-s** [lsc-project/lsc#96]
I've noticed that bin/hsqldb listens on all interfaces by default. This is really surprising given that there is no password (or option to set one), and can lead to information disclosure.

To verify:
`bin/hsqldb --start && ss -ltpn | grep java`

As a minimum, I suggest making the following change in bin/hsqldb:

```
-    -database.0 file:$HSQLDB_DIR/mydb -dbname.0 lscdb > /dev/null 2>&1 &
+    -database.0 file:$HSQLDB_DIR/mydb -dbname.0 lscdb -address 127.0.0.1 > /dev/null 2>&1 &
```

It's also possible to avoid using the server mode, as far as I can tell it's not really necessary for most use cases. I have written a python wrapper script which takes one or more CSV files as an argument, makes a temporary hsqldb text database on disk, runs lsc and then deletes the temporary files. Let me know if you're interested, and I can share the script.

**Comment by coudot** [comment]
For CSV, we also wrote some scripts that can be used with exectuable plugin: https://lsc-project.org/documentation/plugins/executable/perl_csv2ldif

**Comment by coudot** [comment]
Merged on 2.1 and applied to master with cherry-pick
